### PR TITLE
Add frontend components page to manual

### PR DESCRIPTION
**Pull request by fofr**
Detail what components are, where principles and conventions live and where to find component guides.

### CODE REVIEW

File: source/manual/components.html.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+Find components in these guides:
+
+* [static component guide](https://govuk-static.herokuapp.com/component-guide/)
+* [government-frontend component guide](https://government-frontend.herokuapp.com/component-guide/)
+
+## Building components
+
+A component must:
+* [meet the component principles](https://github.com/alphagov/govuk_publishing_components/blob/master/docs/component_principles.md)
```
Comment (andysellick) [inline source/manual/components.html.md]: I think there needs to be an empty line here - it's not rendering the list properly.

File: source/manual/components.html.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+Components are packages of template, style, behaviour and documentation that live in your application or in static.
+
+## Component guides
+
+Components in applications are documented in component guides using the [govuk_publishing_components gem](https://github.com/alphagov/govuk_publishing_components). This mounts a component guide at the path `/component-guide` in development, test and on Heroku review apps.
+
+Find components in these guides:
+
+* [static component guide](https://govuk-static.herokuapp.com/component-guide/)
```
Comment (andysellick) [inline source/manual/components.html.md]: This link doesn't work yet - I assume that's okay?

Comment (fofr) [reply source/manual/components.html.md]: It'll work now.

File: source/manual/components.html.md (hunk 1 above)
Comment (fofr) [reply source/manual/components.html.md]: Fixed :+1:

**Pull request merged**